src/libcommon/log/sentry/ptraces.h

```diff
@@ -0,0 +1,315 @@
+ | PTraces::Basic::AppStart().start();
+ | PTraces::Basic::AppStart().stop();
+ is equivalent to (but less efficient):
+ | PTraces::Basic::AppStart perfMonitor();
+ | perfMonitor.start();
+ | perfMonitor.stop();
+ */
+namespace Basic {
+struct AppStart : public AbstractPTrace {
+        [[nodiscard]] AppStart() : AbstractPTrace({"AppStart", "Strat the application", PTraceName::AppStart}) {}
+};
+
+struct Sync : public AbstractPTrace {
+        [[nodiscard]] explicit Sync(int dbId) :
+            AbstractPTrace({"Synchronisation", "Synchronisation initialization", PTraceName::Sync}, dbId){};
+};
+
+struct UpdateDetection1 : public AbstractPTrace {
+        [[nodiscard]] explicit UpdateDetection1(int dbId) :
+            AbstractPTrace({"UpdateDetection1", "Compute FS operations", PTraceName::UpdateDetection1, PTraceName::Sync}, dbId) {}
+};
+
+struct UpdateDetection2 : public AbstractPTrace {
+        [[nodiscard]] explicit UpdateDetection2(int dbId) :
+            AbstractPTrace({"UpdateDetection2", "UpdateTree generation", PTraceName::UpdateDetection2, PTraceName::Sync}, dbId) {}
+};
+
+struct Reconciliation1 : public AbstractPTrace {
+        [[nodiscard]] explicit Reconciliation1(int dbId) :
+            AbstractPTrace({"Reconciliation1", "Platform inconsistency check", PTraceName::Reconciliation1, PTraceName::Sync},
+                           dbId){};
+};
+
+struct Reconciliation2 : public AbstractPTrace {
+        [[nodiscard]] explicit Reconciliation2(int dbId) :
+            AbstractPTrace({"Reconciliation2", "Find conflicts", PTraceName::Reconciliation2, PTraceName::Sync}, dbId) {}
+};
+
+struct Reconciliation3 : public AbstractPTrace {
+        [[nodiscard]] explicit Reconciliation3(int dbId) :
+            AbstractPTrace({"Reconciliation3", "Resolve conflicts", PTraceName::Reconciliation3, PTraceName::Sync}, dbId) {}
+};
+
+struct Reconciliation4 : public AbstractPTrace {
+        [[nodiscard]] explicit Reconciliation4(int dbId) :
+            AbstractPTrace({"Reconciliation4", "Operation Generator", PTraceName::Reconciliation4, PTraceName::Sync}, dbId) {}
+};
+
+struct Propagation1 : public AbstractPTrace {
+        [[nodiscard]] explicit Propagation1(int dbId) :
+            AbstractPTrace({"Propagation1", "Operation Sorter", PTraceName::Propagation1, PTraceName::Sync}, dbId) {}
+};
+
+struct Propagation2 : public AbstractPTrace {
+        [[nodiscard]] explicit Propagation2(int dbId) :
+            AbstractPTrace({"Propagation2", "Executor", PTraceName::Propagation2, PTraceName::Sync}, dbId) {}
+};
+} // namespace Basic
+
+/* Scoped performance traces will automatically start when the object is created and stop when the object is
+ destroyed.
+ Some scoped performance traces expect to be manually stopped. In this case, the stop() method must be called, else the trace will
+ be consider as aborted when the object will be destroyed. Such traces are indicated by a comment in their class definition.
+ */
+namespace Scoped {
```

Comment:
We reserve uppercase for struct and class names.

Reply:
Done here: [36c844d](https://github.com/Infomaniak/desktop-kDrive/pull/403/commits/36c844d03aca707b493d8b2b0510e39761a25439)